src/cards.scss

```diff
@@ -0,0 +1,139 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  width: 1200px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    color: $main-acent;
+    margin-bottom: 4px;
+    margin-top: 0;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &__price-value {
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $main-acent;
+  }
+
+  &__price-header {
+    font-size: 12px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    margin-top: 0;
+    height: 16px;
+    width: 166px;
+  }
+
+  &__review-count {
+    text-align: right;
+    color: $main-acent;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: $btn-white-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    box-sizing: border-box;
+    border: 1px solid $btn-turquoise-color;
+    border-radius: 5px;
+    background: $btn-turquoise-color;
+
+    &:hover {
+      background: $btn-white-color;
+      color: $btn-turquoise-color;
+    }
+  }
+}
+
+.card:hover .card__title {
+  color: #34568b;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n + 1) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
Similar to the previous comment, ensure that the path to star-active.svg is relative and does not start with a slash.